### PR TITLE
Add ibmjava8 as build target for travis

### PR DESCRIPTION
**Pull request by JLLeitschuh**
### Contributor Checklist
- [x] Base the PR against the `develop` branch
- [x] [Sign Gradle CLA](http://gradle.org/contributor-license-agreement/)
- [x] Provide integration tests to verify changes from a user perspective
- [x] Provide unit tests to verify logic
- [x] Ensure that tests pass locally: `./gradlew check --parallel`

### CODE REVIEW

**Comment by bamboo** [comment]
Hey @JLLeitschuh, is this really supported by Travis?

**Comment by JLLeitschuh** [comment]
It will be very soon.
https://github.com/travis-ci/packer-templates/pull/527#issuecomment-335267361

I'm waiting for @bogdanap to get back to me when there is a stable build of the image that I can pull.

**Comment by bamboo** [comment]
I see 👍

**Comment by bogdanap** [comment]
@JLLeitschuh The new build images that include IBM Java 8 have just been published, so they should be ready for testing.

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@ matrix:
         apt:
           packages:
             - oracle-java9-installer # Forces use of even newer JDK 9 build
+    - os: linux
+      sudo: false
```
Comment (bogdanap) [inline .travis.yml]: Currently, the `group` key below can only be used for `sudo: required`, so this will need to be updated.

Comment (JLLeitschuh) [reply .travis.yml]: Oh, cool. Thank you so much for reviewing this!!! :D

**Comment by JLLeitschuh** [comment]
@bamboo & @eskatos Do either of you have any thoughts on why this failed on J9?

Does the master Gradle repo normally get tested against J9 in your teamcity builds?

**Comment by JLLeitschuh** [comment]
Giving this another shot now that travis is stable for all other builds thanks to #953 by @SUPERCILEX

**Comment by JLLeitschuh** [comment]
Bahaha, this is quite an old PR. Thanks for sweeping up @eskatos

**Pull request closed without merging**